interfaces/Portalicious/src/app/components/page-layout/components/payment-header/payment-header.component.html

```diff
@@ -0,0 +1,16 @@
+<app-page-layout-title-and-actions>
+  <ng-container page-layout-title>
+    @if (payments.isPending()) {
+      <p-skeleton
+        width="24rem"
+        height="1.6rem"
+      />
```

Comment:
This `if` is the same as in `registration-header`.

Maybe we can refactor this to avoid this repetition too. An idea would be to add an `isPending` prop to `app-page-layout-title-and-actions`, and then render the skeleton inside that component instead.

Then we can remove this if and the same if from registration header.